osu.Game/Screens/Menu/MainMenu.cs

```diff
@@ -184,5 +208,18 @@ protected override bool OnKeyDown(InputState state, KeyDownEventArgs args)
 
             return base.OnKeyDown(state, args);
         }
+
+        protected override bool OnKeyUp(InputState state, KeyUpEventArgs args)

```

Comment:
Consider to use `IKeyBindingHandler` with so user can change "Exit" keybindings from Esc

Reply:
I don't think this is necessary. What else would you want *Esc* to do?

Reply:
Wrong way around.. this would mean that you can exit the game with other keys than Escape. But I agree, I don't see why you'd want to use any other key to exit the game.. Not sure if we want this to be supported